example/raw/generate_feed_from_raw_iocs.py

```diff
@@ -1,13 +1,5 @@
 # stdlib imports
-import re
-import sys
-import time
-import urllib
-import json
-import optparse
-import socket
-import base64
-import hashlib
+import re , sys , time ,  json , optparse , socket , base64 , hashlib
```

Comment:
Please revert back to the one import-per-line format as that's in line with Python's style guides: http://pep8.org/#imports

We should remove urllib and json since they're not used in this script, feel free to remove them from the import list